cmd/server/main.go

```diff
@@ -25,17 +24,18 @@ import (
 // @name bux-auth-xpub
 func main() {
 
+	defaultLogger := logging.GetDefaultLogger()
 	// Load the Application Configuration
 	appConfig, err := config.Load("")
 	if err != nil {
-		logger.Fatalf(dictionary.GetInternalMessage(dictionary.ErrorLoadingConfig), err.Error())
+		defaultLogger.Error().Msgf(dictionary.GetInternalMessage(dictionary.ErrorLoadingConfig), err.Error())
```

Comment:
Is it possible to use `zerolog.fatal` ?

Reply:
My mistake, done

Reply:
Please check other `fatal` cases in this file. `fatal` will shutdown the program. `error` only show a message

Reply:
Done